tests/tiles/tiles.spec.ts

```diff
@@ -590,26 +590,66 @@ describe('Snap a bounding box to tile grid', () => {
   });
 });
 
-describe('Find minimal zoom that can contain bounding box in one tile', () => {
-  it('should return correct zoom starting at zoom 2 and moving to 1', () => {
+describe('#minimalBoundingTile', () => {
+  it('should return correct tile starting lookup at zoom 1 and moving to 0', () => {
     const boundingBox = new BoundingBox(-135, -45, -45, 45);
-    const minimalZoom = findMinimalZoom(boundingBox);
-    const expectedZoom = 1;
-    expect(minimalZoom).toEqual(expectedZoom);
+    const expectedTile = new Tile(0, 0, 0, 1);
+
+    const tile = minimalBoundingTile(boundingBox);
+
+    expect(tile).toStrictEqual(expectedTile);
+  });
+
+  it('should return correct tile starting at lookup at zoom 2 and moving to 0', () => {
```

Comment:
This test is the same as the previous one, why not pass parameters to a unified test?